pkg/tsdb/elasticsearch/response_parser.go

```diff
@@ -51,12 +50,13 @@ func parseResponse(ctx context.Context, responses []*es.SearchResponse, targets
 	}
 
 	for i, res := range responses {
+		start := time.Now()
 		target := targets[i]
 
 		if res.Error != nil {
 			mt, _ := json.Marshal(target)
 			me, _ := json.Marshal(res.Error)
-			logger.Error("Error response from Elasticsearch", "error", string(me), "query", string(mt))
+			logger.Debug("Processing error response from Elasticsearch", "error", string(me), "query", string(mt))
```

Comment:
Loki is logging this as an error: https://github.com/grafana/grafana/blob/main/pkg/tsdb/loki/api.go#L205